test_projects/flutter/flutter_example/build_and_run_tests_firebase

```diff
@@ -0,0 +1,16 @@
+flutter build apk
```

Comment:
should not this file have `.sh` extension?

Reply:
Good point! Changed